Bluepill-runner/BluepillRunnerTests/BPIntegrationTests.m

```diff
@@ -110,7 +110,7 @@ - (void)testTwoBPInstancesWithXCTestRunFile {
     self.config.testBundlePath = nil;
     self.config.testRunnerAppPath = nil;
     NSString *runtime = [[NSString stringWithUTF8String:BP_DEFAULT_RUNTIME] stringByReplacingOccurrencesOfString:@"iOS " withString:@""];
-    NSString *xcTestRunFile = [NSString stringWithFormat:@"BPSampleApp_iphonesimulator%@-x86_64.xctestrun", runtime];
+    NSString *xcTestRunFile = [NSString stringWithFormat:@"Build/Products/BPSampleApp_iphonesimulator%@-x86_64.xctestrun", runtime];
```

Comment:
why is this changed?

Reply:
Because BluepillLib is moved to different location and the base path is updated to ../../ to that xctestrun file.